src/types/chat/action.rs

```diff
@@ -15,10 +15,10 @@ pub enum Action {
     RecordVideo,
     /// About to send a video.
     UploadVideo,
-    /// About to send a generated audio.
-    RecordAudio,
-    /// About to send an audio.
-    UploadAudio,
+    /// About to send a generated voice-audio.
```

Comment:
```suggestion
    /// About to send a generated voice.
```
(or `a generated voice message` if `voice` on its own sounds a bit awkward)

Reply:
tbh, I disagree with all these "About to `send`..". can't we rename those to "Status: sending/recording"